crates/forge/src/scarb.rs

```diff
@@ -213,6 +223,41 @@ pub fn dependencies_for_package(
     Ok(dependencies)
 }
 
+fn validate_fork_config(config: Result<ForgeConfig>) -> Result<ForgeConfig> {
+    if let Ok(ForgeConfig {
+        fork: Some(forks), ..
+    }) = &config
+    {
+        let names: Vec<String> = forks.iter().map(|fork| fork.name.clone()).collect();
+        let removed_duplicated_names: HashSet<String> = names.iter().cloned().collect();
```

Comment:
```suggestion
        let removed_duplicated_names: Vec<String> = names.clone().into_iter().unique().collect();
```

This way we don't create a HashSet. But it doesn't really optimize anything most likely.